Assets/Scripts/Characters/Human/Horse/Horse.cs

```diff
@@ -20,11 +21,22 @@ class Horse: BaseCharacter
         private float _idleTimeLeft;
         private float _teleportTimeLeft;
         private float _jumpCooldownLeft;
+
+        [SerializeField]
+        public Transform PassengerSeat;
+        public bool _hasPassenger = false;
         
         public void Init(Human human)
         {
             base.Init(true, human.Team);
             _owner = human;
+            photonView.RPC("SetOwnerRPC", RpcTarget.AllBuffered, human.photonView.ViewID);
```

Comment:
Where possible, try to use `nameof` instead of magic strings. As this will make the code more maintainable. Plus you can easily see then which RPCs use a specific method.

So instead, it would be 
```csharp
            photonView.RPC(nameof(SetOwnerRPC), RpcTarget.AllBuffered, human.photonView.ViewID);
```